src/modules/Registration/index.scss

```diff
@@ -0,0 +1,81 @@
+@import "./src/styles/color-palette";
+
+.container {
+  margin-top: 3.5rem;
+  padding: 3rem;
+  border-radius: $border-radius;
+
+  .success-msg {
+    color: $primary-color;
+    background: $color-green;
+    display: inline-block;
+    width: 100%;
+    text-align: center;
+    padding: 0.5rem;
+    border-radius: $border-radius;
+    margin-bottom: 1rem;
+    font-size: 1.2rem;
+    font-weight: 600;
+  }
+
+  h1 {
```

Comment:
Rewrite this file's class names with correct BEM. And use class names instead of usual tags.